lib/venue.rb

```diff
@@ -1,2 +1,38 @@
 class Venue
+  attr_accessor :patrons
+  attr_reader :name, :capacity
+  def initialize(name, int)
```

Comment:
Something to keep in mind here, would someone know what `int` was referencing here? Maybe a more descriptive name could be capacity_int?